libcontainerd/shimopts/convert.go

```diff
@@ -17,7 +17,7 @@ func Generate(runtimeType string, opts map[string]interface{}) (interface{}, err
 	// options in the same way.
 	var out interface{}
 	switch runtimeType {
-	case plugin.RuntimeRuncV1, plugin.RuntimeRuncV2:
+	case plugin.RuntimeRuncV2:
```

Comment:
?

Reply:
I'll need to rebase things a bit backwards. Modern containerd no longer has runc v1.

Reply:
containerd 1.7 still has RuncV1; what you are describing as "modern" containerd is an unreleased development branch.

Reply:
Yeah, I do understand that. Gimme some time to polish this stuff :D It is currently extracted as-is from https://github.com/macOScontainers/moby where I had to use newer containerd because of dependency on FreeBSD patches.